VIDEO_1_SCRIPT.md

```diff
@@ -0,0 +1,99 @@
+
+- **Web services APIs**: these are the application development interfaces that enable the exchange of information between a web service (software that provides access to a particular service via a URL) and an application. 
+
+    There are four types of Web services APIs:
+
+    •	SOAP (Simple Object Access Protocol), a standard protocol for exchanging information and data in XML between two objects.
+
+    •	XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.
+
+    • JSON-RPC, the same protocol but in JSON format. 
+
+    •	REST (Representational State Transfer), a software architecture for hypermedia systems on the World Wide Web; a REST API uses the HTTP protocol.
+
+- **Library-based APIs**: these types of APIs are those that allow an application to import a library from other software to exchange information. Nowadays, most of the libraries that provide access to products and services are designed in JavaScript.
+
+- **Class-based APIs**: this type of application development interfaces allows the connection with data around classes, as is usual in object-oriented programming with Java. The Java API uses abstract classes for the creation of applications just like any program developed in this language. These classes provide everything needed to perform all kinds of functions within these applications. The Java development interface is organized into packages and each of those packages in turn contains a set of related classes.  
+
+- **Function APIs in operating systems**: software programs are continuously interacting with operating systems. In many cases, the way they do so is through APIs. Operating systems such as Windows have APIs that enable such communication between programs and the OS.
+
+## Benefits os APIs
+
+There are many benefits of APIs, it can be highlighted:
+
+- **Adaptation**: needs change over time and APIs help to anticipate changes. When working on this technology, data migration can be better supported, while offering a better review of the information. 
+
+- **Automation**: APIs allow IT teams rather than people to do the work. Through APIs, workflows can be updated so that they can be done in less time while being more productive.
+
+- **Application**: enabling API access to app components provides flexibility in the delivery of services and information.  
+
+- **Customization**: through APIs, any user can customize content and services.
+
+- **Efficiency**: when access to an API is provided, the content that is generated can be automatically published and made available to all channels. It allows it to be shared and distributed more easily. 
+
+- **Integration**: APIs allow content to be more easily embedded from any site or application. This ensures seamless information delivery and an integrated user experience.
+
+## Example of an API
+
+I think it would be great to give some example of API related to AsyncAPI, any idea or suggestion?
```

Comment:
Hmm.. the problem is that this script is already long, so a demo of an example API seems like a video of its own. 🤔  Unless this video is 10+ mins long, I guess I'm unsure as to what the goal for the 1st video is 😀

Reply:
The idea of the first video is to inform about what an API is and to make it a video with complete information about it. The proposal is a video lasting at most 15 minutes and then make a short version. 

I don't know how long it takes to make a demo of an API 😅. We could include it in this script and make it a video of about 15 minutes and then make two teasers. One with the theoretical part and one with the practical part. What do you think? Do you think it is better to do it in a separate script? or should the theoretical script be shorter?

Reply:
got it! for sure, I am sure one of the other contributors can contribute the API demo and plan it to only be a few mins long :)

Hmm, if the API demo is only a few mins long, (say 3mins), then I would include it in both the short and long version. But anything longer than that and I would prob make two separate short videos... one could be the conceptual one and the other could be just a standalone demo... 

Curious to see what others end up thinking or suggesting too!

Reply:
Sounds good, it makes sense.

Reply:
I agree it's already long and packed with lots of information. I think a demo may deserve its own video so we're not that constrained. Especially, because many (most?) people learn by example.